emulators/oberon-risc/files/patch-rename-program.diff

```diff
@@ -0,0 +1,30 @@
+--- a/Makefile
++++ b/Makefile
+@@ -13,7 +13,7 @@ RISC_SOURCE = \
+ 	src/raw-serial.c src/raw-serial.h \
+ 	src/sdl-clipboard.c src/sdl-clipboard.h
+ 
+-risc: $(RISC_SOURCE)
++oberon-risc: $(RISC_SOURCE)
+ 	$(CC) -o $@ $(filter %.c, $^) $(RISC_CFLAGS)
+ 
+ # Assumes SDL2 framework download, following README instructions for install.
+@@ -22,4 +22,4 @@ osx: $(RISC_SOURCE)
+ 		-I  /Library/Frameworks/SDL2.framework/Headers/
+ 
+ clean:
+-	rm -f risc
++	rm -f oberon-risc
+diff --git a/src/sdl-main.c b/src/sdl-main.c
+index b82b291..ce2c835 100644
+--- a/src/sdl-main.c
++++ b/src/sdl-main.c
+@@ -83,7 +83,7 @@ static void fail(int code, const char *fmt, ...) {
+ }
+ 
+ static void usage() {
+-  puts("Usage: risc [OPTIONS...] DISK-IMAGE\n"
++  puts("Usage: oberon-risc [OPTIONS...] DISK-IMAGE\n"
```

Comment:
MacOS has `getprogname(3)`, which might be cleaner than hard coding this, though that's a matter of taste. You can also use `xinstall` to do the rename for you so you don't have to change the Makefile.